### PR TITLE
[PATCH v2] configure.ac: update API and .so version to v1.18.0.0

### PR DESCRIPTION
**Pull request by muvarov**
Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -36,10 +36,10 @@ AM_SILENT_RULES([yes])
 # 3. if interfaces were removed, then use C+1:0:0
 ##########################################################################
 
-ODP_LIBSO_VERSION=117:0:0
+ODP_LIBSO_VERSION=118:0:0
 AC_SUBST(ODP_LIBSO_VERSION)
 
-ODPHELPER_LIBSO_VERSION=113:2:1
+ODPHELPER_LIBSO_VERSION=118:0:0
```
Comment (lumag) [inline configure.ac]: Helper is compatible, as far as I understand, so we should not bump major version here.

Comment (Bill-Fischofer-Linaro) [reply configure.ac]: Perhaps, but as this is a new LTS release might be best to sync everything? Apps are going to be recompiling for Tiger Moth anyway if they're coming from Monarch.

Comment (lumag) [reply configure.ac]: @Bill-Fischofer-Linaro soname is not about releases, but about compatibility, so I vote for 114:0:2 for odphelper.

**Comment by muvarov** [comment]
v2: change helper .so name to 14:0:2

**Comment by lumag** [comment]
Reviewed-by: Dmitry Eremin-Solenikov

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**